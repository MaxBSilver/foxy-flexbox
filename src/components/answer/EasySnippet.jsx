import React from "react";

export default function EasySnippet(props) {
  return (
    <React.Fragment>
      <div className="input-one">
        <p className="code-prompt"> justify-content : </p>
        <input
          type="text"
          className="search-input"
          name="jcVal"
          value={props.jcVal}
          onChange={props.handleChange}
          placeholder=""
        />
      </div>
      <div className="input-two">
        <p className="code-prompt"> align-content : </p>
        <input
          type="text"
          className="search-input"
          name="acVal"
          value={props.acVal}
          onChange={props.handleChange}
          placeholder=""
        />
      </div>
    </React.Fragment>
  );
}
