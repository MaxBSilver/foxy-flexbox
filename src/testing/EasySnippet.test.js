import React from 'react';
import EasySnippet from '../components/answer/EasySnippet';
import { shallow } from 'enzyme';
describe('EasySnippet', () => {
    let wrapper;
  
    beforeEach(() => {
      wrapper = shallow(
        <EasySnippet
        />
      );
    });
    it('should match the snapshot with all data passed in', () => {
        expect(wrapper).toMatchSnapshot();
      });
  });