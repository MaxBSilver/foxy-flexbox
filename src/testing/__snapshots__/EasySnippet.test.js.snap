// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EasySnippet should match the snapshot with all data passed in 1`] = `
<Fragment>
  <div
    className="input-one"
  >
    <p
      className="code-prompt"
    >
       justify-content : 
    </p>
    <input
      className="search-input"
      name="jcVal"
      placeholder=""
      type="text"
    />
  </div>
  <div
    className="input-two"
  >
    <p
      className="code-prompt"
    >
       align-items : 
    </p>
    <input
      className="search-input"
      name="acVal"
      placeholder=""
      type="text"
    />
  </div>
</Fragment>
`;
